react/src/components/Form/SelectField.tsx

```diff
@@ -7,6 +7,7 @@ export interface IDropdownOption extends OptionBase {
   value: string;
   label: string;
   color?: string | undefined | null;
+  data?: object | undefined | null;
```

Comment:
the QuestionMark should be enough. not sure why you need to define the case `| undefined | null`
btw it is bad practise to use null. I try mostly to only use undefined. This can be sometimes tough though.

https://medium.com/@o_o_o/null-vs-undefined-can-i-use-only-one-a3b7db5468f2